README.md

```diff
@@ -3,6 +3,52 @@ React Native Tipsi custom UI elements
 
 ## Components
 
+### `<ColoredTabs />`
+
+Customizable colored tabs.
+
+#### ColoredTabs Props
+
+| Name | Desc | Type | Default
+| --- | --- | --- | --- |
+| `selected` | id of selected tab.  | String | `undefined`
+| `onPress` | Handle press on tab action. Receive id of selected tab as argument. | Function | `undefined`
+
+
+#### ColoredTabs.Item Props
+
+| Name | Desc | Type | Default
+| --- | --- | --- | --- |
+| `name` | Text of tab. | String | `undefined`
+| `color` | Color of tab | String | `undefined`
+| `id` | Tab id. This id will passed to `onPress` for identify action. | String | `undefined`
+| `active` | Show item as active  | Boolean | `false`
+| `onPress` | Handle press action | Function | `undefined`
+
+#### Example
+
+
+```js
+import React from 'react'
+import { ColoredTabs } from 'tipsi-ui-kit'
+
+handlePress = (id) => (console.log(`Tab with id ${id} is clicked!`)
+
+const Example = () => (
+    <ColoredTabs onPress={this.handlePress}>
+      <ColoredTabs.Item id="0" name="One" color="crimson" />
+      <ColoredTabs.Item id="1" name="Two" color="orange" />
+      <ColoredTabs.Item id="2" name="Three" color="chartreuse" />
+      <ColoredTabs.Item id="3" name="Four" color="dodgerblue" />
+    < /ColoredTabs>
+)
```

Comment:
Also please check eslint-plugin in your editor. We use **2 spaces** as `tab` length